app/scripts/directives/editPvc.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .directive("editPvc",
+             function($uibModal,
+                      $filter,
+                      $routeParams,
+                      APIService,
+                      DataService,
+                      ProjectsService,
+                      NotificationsService,
+                      Logger) {
+
+    return {
+      restrict: "E",
+      scope: {
+        // pvc object
+        pvc: "@",
+        // Alerts object for success and error alerts.
+        alerts: "=",
+        // Optional link label. Defaults to "Expand".
+        label: "@?",
+        // Optional redirect URL when the delete succeeds
+        redirectUrl: "@?",
+        // currentCapacity of the pvc
+        currentCapacity: "@"
```

Comment:
Why does this need to be passed in as an attribute? Isn't it already in the PVC?

Reply:
ACK